src/manage/project-history/index.md

```diff
@@ -3,16 +3,34 @@
 
 Each time someone synchronises the project from a mobile device or from QGIS through <QGISPluginName />, a new project version will be created. Thus, it is possible to see the details of the changes made in <MainPlatformNameLink /> project by different users or devices. 
 
-On <AppDomainNameLink />, you can view what files have been added or removed. If you use GeoPackage for your survey, you can also see the list of the features which have been added, deleted or updated.
+Using the <QGISPluginNameShort /> or the <DashboardShortLink />, you can view what files have been added or removed. If you use GeoPackage for your survey, you can also see the list of the features which have been added, deleted or updated.
 
 
-:::tip
+:::tip Project history in QGIS
 Great tools to inspect the project history and compare changes between project versions can be found in the <QGISPluginNameShort />:
-- use [Create report](./plugin/#create-report-processing-toolbox) to get an overview of changes for a range of project versions.
-- use [Create diff](./plugin/#create-diff-processing-toolbox) to extract changes made between two versions of a layer
+- use [Project history](#project-history-in-qgis) to explore changes in the **Changes Viewer**
+- use [Create report](../plugin/#create-report-processing-toolbox) to get an overview of changes for a range of project versions.
+- use [Create diff](../plugin/#create-diff-processing-toolbox) to extract changes made between two versions of a layer
 :::
 
-## Project history and advanced changelog
+## Project history in QGIS
+Project history and version can be inspected using the <QGISPluginName /> by clicking the **Project history** button of the toolbar.
+![Project history button](../plugin/plugin-toolbar-project-history.jpg "Project history button")
+
+The button opens the **Changes Viewer** window. Here, all available project versions are listed in the left panel. 
+
```

Comment:
Would be nice to say that this shows changes only of the selected version, not the state of the project at that version.